.github/workflows/stale-items.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: 'Stale-Items'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Comment:
See previous comment on cron jobs. Document what the schedule is as well as why that's the schedule.